docs/src/topics/flavors/dns-loadbalancing.md

```diff
@@ -0,0 +1,42 @@
+# DNS based apiserver Load Balancing
+
+This flavor configures DNS records that resolve to the public (ipv4 and/or IPv6) IPs of the control plane nodes where the apiserver pods are running. No NodeBalancer will be created.
+This needs the following to be set in the `LinodeCluster` spec under `network`
+```bash
+kind: LinodeCluster
+spec:
+    network:
+        loadBalancerType: dns
+        dnsRootDomain: test.net
+        dnsUniqueIdentifier: abc123
+```
+Along with this, the `test.net` domain needs to be registered and also be pre-configured as a domain on Linode CM. Using the `LINODE_DNS_TOKEN` env var, you can pass the API token of a different account if the Domain has been created in another acount under Linode CM
+
+With these changes, the controlPlaneEndpoint is set to `<domain-name>-<uniqueid>.<root-domain>`. This will set as the server in the KUBECONFIG as well.
+The controller will create A/AAAA and TXT records under the domain in Linode CM
+
+
+## Specification
+| Supported Control Plane | CNI    | Default OS   | Installs ClusterClass | IPv4 | IPv6 |
+|-------------------------|--------|--------------|-----------------------|------|------|
+| kubeadm                 | Cilium | Ubuntu 22.04 | No                    | Yes  | Yes  |
```

Comment:
This should be false for ipv6 since the flavor base isn't dual-stack right?

Reply:
It is now after [this PR](https://github.com/linode/cluster-api-provider-linode/pull/383)

Reply:
Ah gotcha, I was looking at `templates/flavors/kubeadm/dns-loadbalancing/kustomization.yaml`